src/app/contextBroker/contextBroker.cpp

```diff
@@ -888,8 +888,6 @@ static void logEnvVars(void)
   }
 }
 
-
-
```

Comment:
Here you seem to have removed two empty lines.
I'd like them back.
Throughout Orion source code, we try to maintain three empty line between functions.
It makes the file easier to read.

Reply:
Fixed in 17364b1